test/CMakeLists.txt

```diff
@@ -5,8 +5,12 @@ set_target_properties(gmock PROPERTIES FOLDER "Dependencies")
 set_target_properties(gmock_main PROPERTIES FOLDER "Dependencies")
 
 # common things that the tests need
+set(TEST_SRCS test.h test.cc)
+if(ENABLE_DATA_TOOLS)
+  list(APPEND TEST_SRCS gurka/gurka.h gurka/gurka.cc)
+endif()
```

Comment:
no gurka either unless you enable data_tools (gurka builds tiles of course which is what mjolnir does)